### PR TITLE
HW4 Task04

### PR DESCRIPTION
**Pull request by LinaKrevetochka**

### CODE REVIEW

File: hw/hw_04_task_04.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# flake8: noqa: CCR001
+def fizzbuzz(n: int) -> List[str]:
+    """
+    Function takes a number N as an input and returns N FizzBuzz numbers.
+
+    Instruction:
+     - Install Python 3.8 (https://www.python.org/downloads/)
+     - Install pytest `pip install pytest`
+     - Clone the repository <path your repository>
+     - Checkout branch <your branch>
```
Comment (Cjkjvfnby) [inline hw/hw_04_task_04.py]: `git checkout your <your branch>` does not work for me.

File: hw/hw_04_task_04.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+# flake8: noqa: CCR001
+def fizzbuzz(n: int) -> List[str]:
+    """
+    Function takes a number N as an input and returns N FizzBuzz numbers.
+
+    Instruction:
+     - Install Python 3.8 (https://www.python.org/downloads/)
+     - Install pytest `pip install pytest`
```
Comment (Cjkjvfnby) [inline hw/hw_04_task_04.py]: Why do I need pytes?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hw/hw_04_task_04.py (hunk 2 above)
Comment (LinaKrevetochka) [reply hw/hw_04_task_04.py]: I don't, my bad.

Comment (Cjkjvfnby) [reply hw/hw_04_task_04.py]: Why `pytest` was in the instruction template?

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**